grpc-proto/proto/stargate.proto

```diff
@@ -29,5 +30,11 @@ service Stargate {
 
   // Executes a batch of CQL queries.
   rpc ExecuteBatch(Batch) returns (Response) {}
+
+  // Schema creation
+  rpc ExecuteSchemaOperation(CqlSchemaOperation) returns (Response) {}
```

Comment:
What do you think about specific RPC calls for creating each type of schema?

```grpc
rpc CreateKeyspace(...) returns (...) {}
rpc CreateTable(...) returns (...) {}
rpc CreateType(...) return (...) {}
```

Reply:
that would be simpler in some ways. I was trying to emulate the style of ExecuteQuery/ExecuteBatch, but am fine breaking them out